docs/kubernetes_distros.md

```diff
@@ -22,6 +22,10 @@ Google's GKE hosted Kubernetes platform enables RBAC by default. You will need t
 
 See [Tiller and role-based access control](https://docs.helm.sh/using_helm/#role-based-access-control) for more information.
 
+## AKS
+
+Helm works with [Azure Kubernetes Service](https://docs.microsoft.com/en-us/azure/aks/kubernetes-helm). If using an RBAC-enabled AKS cluster, you need [a service account and role binding for the Tiller service](https://helm.sh/docs/using_helm/#tiller-namespaces-and-rbac).
```

Comment:
would it be better to link to the Azure documentation on this subject? The docs there may be better suited for the cloud provider's particular setup, whereas the RBAC doc in Helm tries to explain it at a high level.

https://docs.microsoft.com/en-us/azure/aks/kubernetes-helm#create-a-service-account

Reply:
Good call! I've made this change.